.github/ct-install.yaml

```diff
@@ -12,6 +12,7 @@ chart-dirs:
 excluded-charts:
   - charts/dependency/subchart
   - charts/incubator/twingate-connector
+  - charts/incubator/eco
```

Comment:
?

Reply:
the web server never starts in the CI, tested locally and it works.